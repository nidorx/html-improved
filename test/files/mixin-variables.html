<mixin name="mixin-test">
    <vars
        varOne="value in mixin"
        />

    <p>#{varOne}</p>
</mixin>

<p>#{varOne}</p>

<vars
    varOne="value out of mixin"
    />

<mixin-test />

