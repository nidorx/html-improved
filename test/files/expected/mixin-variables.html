<p>value out of mixin</p>
<p>value in mixin</p>

